src/webserver.cpp

```diff
@@ -454,14 +457,61 @@ MHD_Result webserver::post_iterator(void *cls, enum MHD_ValueKind kind,
         const char *transfer_encoding, const char *data, uint64_t off, size_t size) {
     // Parameter needed to respect MHD interface, but not needed here.
     std::ignore = kind;
-    std::ignore = filename;
     std::ignore = content_type;
     std::ignore = transfer_encoding;
```

Comment:
While re-reading this change I was wonder whether we should include these two parameters as part of the `file_info` structure as well to make it possible for the clients to know the content type and encoding of the file. What do you think?

It would likely also align with the issue that originated the whole initial discussion: https://github.com/etr/libhttpserver/issues/229

Reply:
Done, added content_type and transfer_encoding.

One thing to note:
These two are only available "if known" (as written in libmicrohttpd documentation).
In my tests the transfer_encoding was always nullptr and therefor not set.
The content_type I tested successfully (and integrated in the file_upload example)